stylesheets/style.css

```diff
@@ -5,3 +5,219 @@ main-titles dark-grey: #2C303F;
 paragraph grey: #5b5e6d;
 nav grey: #5b5e6d;
 */
+
+html {
+  box-sizing: border-box;
+}
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  line-height: 150%;
+  font-size: 14px;
+}
+
+ul,
+li {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+h1,
+h2 {
+  font-family: Georgia, "Times New Roman", Times, serif;
+  font-size: 1.75rem;
+}
+
+button {
+  padding: 10px;
+  width: 100%;
+  font-weight: bold;
+  border-radius: 2px;
+}
+
+/* For up to 768px */
```

Comment:
The way that we use responsive web design and media queries is the following:
First always add CSS for the mobile and then with the media query `@media (min-width: 768px)`we style for desktop or bigger screens. So you don't need this `max-width` media query

Reply:
Please if you have questions about this let us know and we can take a look together